2021-09-19 14:25:38.0934|INFO|Info|User Iserial:1144
2021-09-19 14:25:38.1414|ERROR|Info|{"Message":"An exception occurred during the operation, making the result invalid.  Check InnerException for exception details.","Data":{},"InnerException":{"Message":"The HTTP request to 'http://localhost:18559/Service/CRUD_ManagerService.svc' has exceeded the allotted timeout. The time allotted to this operation may have been a portion of a longer timeout.","Data":{},"InnerException":{"Status":6,"Response":null,"Message":"","Data":{},"InnerException":{"Status":6,"Response":null,"Message":"","Data":{},"InnerException":null,"StackTrace":"   at System.Net.Browser.BrowserHttpWebRequest.InternalEndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Browser.BrowserHttpWebRequest.<>c__DisplayClassa.<EndGetResponse>b__9(Object sendState)\r\n   at System.Net.Browser.AsyncHelper.<>c__DisplayClass4.<BeginOnUI>b__0(Object sendState)"},"StackTrace":"   at System.Net.Browser.AsyncHelper.BeginOnUI(SendOrPostCallback beginMethod, Object state)\r\n   at System.Net.Browser.BrowserHttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.ServiceModel.Channels.HttpChannelFactory.HttpRequestChannel.HttpChannelAsyncRequest.CompleteGetResponse(IAsyncResult result)"},"StackTrace":"   at System.ServiceModel.AsyncResult.End[TAsyncResult](IAsyncResult result)\r\n   at System.ServiceModel.Channels.ServiceChannel.EndCall(String action, Object[] outs, IAsyncResult result)\r\n   at System.ServiceModel.ClientBase`1.ChannelBase`1.EndInvoke(String methodName, Object[] args, IAsyncResult result)\r\n   at CCWFM.CRUDManagerService.CRUD_ManagerServiceClient.CRUD_ManagerServiceClientChannel.EndGetGeneric(IAsyncResult result)\r\n   at CCWFM.CRUDManagerService.CRUD_ManagerServiceClient.CCWFM.CRUDManagerService.CRUD_ManagerService.EndGetGeneric(IAsyncResult result)\r\n   at CCWFM.CRUDManagerService.CRUD_ManagerServiceClient.OnEndGetGeneric(IAsyncResult result)\r\n   at System.ServiceModel.ClientBase`1.OnAsyncCallCompleted(IAsyncResult result)"},"StackTrace":"   at System.ComponentModel.AsyncCompletedEventArgs.RaiseExceptionIfNecessary()\r\n   at CCWFM.CRUDManagerService.GetGenericCompletedEventArgs.get_Result()\r\n   at CCWFM.ViewModel.OGViewModels.StyleHeaderViewModel.<GetTblStyleCategory>b__87_0(Object s, GetGenericCompletedEventArgs sv)\r\n   at CCWFM.CRUDManagerService.CRUD_ManagerServiceClient.OnGetGenericCompleted(Object state)"}
2021-09-19 14:25:38.5481|INFO|Info|User Iserial:1144
2021-09-19 14:25:38.5481|ERROR|Info|{"Message":"An exception occurred during the operation, making the result invalid.  Check InnerException for exception details.","Data":{},"InnerException":{"Message":"The HTTP request to 'http://localhost:18559/Service/CRUD_ManagerService.svc' has exceeded the allotted timeout. The time allotted to this operation may have been a portion of a longer timeout.","Data":{},"InnerException":{"Status":6,"Response":null,"Message":"","Data":{},"InnerException":{"Status":6,"Response":null,"Message":"","Data":{},"InnerException":null,"StackTrace":"   at System.Net.Browser.BrowserHttpWebRequest.InternalEndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Browser.BrowserHttpWebRequest.<>c__DisplayClassa.<EndGetResponse>b__9(Object sendState)\r\n   at System.Net.Browser.AsyncHelper.<>c__DisplayClass4.<BeginOnUI>b__0(Object sendState)"},"StackTrace":"   at System.Net.Browser.AsyncHelper.BeginOnUI(SendOrPostCallback beginMethod, Object state)\r\n   at System.Net.Browser.BrowserHttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.ServiceModel.Channels.HttpChannelFactory.HttpRequestChannel.HttpChannelAsyncRequest.CompleteGetResponse(IAsyncResult result)"},"StackTrace":"   at System.ServiceModel.AsyncResult.End[TAsyncResult](IAsyncResult result)\r\n   at System.ServiceModel.Channels.ServiceChannel.EndCall(String action, Object[] outs, IAsyncResult result)\r\n   at System.ServiceModel.ClientBase`1.ChannelBase`1.EndInvoke(String methodName, Object[] args, IAsyncResult result)\r\n   at CCWFM.CRUDManagerService.CRUD_ManagerServiceClient.CRUD_ManagerServiceClientChannel.EndGetTblSizeGroup(Int32& fullCount, IAsyncResult result)\r\n   at CCWFM.CRUDManagerService.CRUD_ManagerServiceClient.CCWFM.CRUDManagerService.CRUD_ManagerService.EndGetTblSizeGroup(Int32& fullCount, IAsyncResult result)\r\n   at CCWFM.CRUDManagerService.CRUD_ManagerServiceClient.OnEndGetTblSizeGroup(IAsyncResult result)\r\n   at System.ServiceModel.ClientBase`1.OnAsyncCallCompleted(IAsyncResult result)"},"StackTrace":"   at System.ComponentModel.AsyncCompletedEventArgs.RaiseExceptionIfNecessary()\r\n   at CCWFM.CRUDManagerService.GetTblSizeGroupCompletedEventArgs.get_Result()\r\n   at CCWFM.ViewModel.OGViewModels.StyleHeaderViewModel.<.ctor>b__32_43(Object s, GetTblSizeGroupCompletedEventArgs sv)\r\n   at CCWFM.CRUDManagerService.CRUD_ManagerServiceClient.OnGetTblSizeGroupCompleted(Object state)"}
2021-09-19 14:25:38.5831|INFO|Info|User Iserial:1144
2021-09-19 14:25:38.5831|ERROR|Info|{"Message":"An exception occurred during the operation, making the result invalid.  Check InnerException for exception details.","Data":{},"InnerException":{"Message":"The HTTP request to 'http://localhost:18559/Service/CRUD_ManagerService.svc' has exceeded the allotted timeout. The time allotted to this operation may have been a portion of a longer timeout.","Data":{},"InnerException":{"Status":6,"Response":null,"Message":"","Data":{},"InnerException":{"Status":6,"Response":null,"Message":"","Data":{},"InnerException":null,"StackTrace":"   at System.Net.Browser.BrowserHttpWebRequest.InternalEndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Browser.BrowserHttpWebRequest.<>c__DisplayClassa.<EndGetResponse>b__9(Object sendState)\r\n   at System.Net.Browser.AsyncHelper.<>c__DisplayClass4.<BeginOnUI>b__0(Object sendState)"},"StackTrace":"   at System.Net.Browser.AsyncHelper.BeginOnUI(SendOrPostCallback beginMethod, Object state)\r\n   at System.Net.Browser.BrowserHttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.ServiceModel.Channels.HttpChannelFactory.HttpRequestChannel.HttpChannelAsyncRequest.CompleteGetResponse(IAsyncResult result)"},"StackTrace":"   at System.ServiceModel.AsyncResult.End[TAsyncResult](IAsyncResult result)\r\n   at System.ServiceModel.Channels.ServiceChannel.EndCall(String action, Object[] outs, IAsyncResult result)\r\n   at System.ServiceModel.ClientBase`1.ChannelBase`1.EndInvoke(String methodName, Object[] args, IAsyncResult result)\r\n   at CCWFM.CRUDManagerService.CRUD_ManagerServiceClient.CRUD_ManagerServiceClientChannel.EndGetGeneric(IAsyncResult result)\r\n   at CCWFM.CRUDManagerService.CRUD_ManagerServiceClient.CCWFM.CRUDManagerService.CRUD_ManagerService.EndGetGeneric(IAsyncResult result)\r\n   at CCWFM.CRUDManagerService.CRUD_ManagerServiceClient.OnEndGetGeneric(IAsyncResult result)\r\n   at System.ServiceModel.ClientBase`1.OnAsyncCallCompleted(IAsyncResult result)"},"StackTrace":"   at System.ComponentModel.AsyncCompletedEventArgs.RaiseExceptionIfNecessary()\r\n   at CCWFM.CRUDManagerService.GetGenericCompletedEventArgs.get_Result()\r\n   at CCWFM.ViewModel.OGViewModels.StyleHeaderViewModel.<.ctor>b__32_6(Object s, GetGenericCompletedEventArgs sv)\r\n   at CCWFM.CRUDManagerService.CRUD_ManagerServiceClient.OnGetGenericCompleted(Object state)"}
2021-09-19 14:25:39.0512|INFO|Info|User Iserial:1144
2021-09-19 14:25:39.0512|ERROR|Info|{"Message":"An exception occurred during the operation, making the result invalid.  Check InnerException for exception details.","Data":{},"InnerException":{"Message":"The HTTP request to 'http://localhost:18559/Service/CRUD_ManagerService.svc' has exceeded the allotted timeout. The time allotted to this operation may have been a portion of a longer timeout.","Data":{},"InnerException":{"Status":6,"Response":null,"Message":"","Data":{},"InnerException":{"Status":6,"Response":null,"Message":"","Data":{},"InnerException":null,"StackTrace":"   at System.Net.Browser.BrowserHttpWebRequest.InternalEndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Browser.BrowserHttpWebRequest.<>c__DisplayClassa.<EndGetResponse>b__9(Object sendState)\r\n   at System.Net.Browser.AsyncHelper.<>c__DisplayClass4.<BeginOnUI>b__0(Object sendState)"},"StackTrace":"   at System.Net.Browser.AsyncHelper.BeginOnUI(SendOrPostCallback beginMethod, Object state)\r\n   at System.Net.Browser.BrowserHttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.ServiceModel.Channels.HttpChannelFactory.HttpRequestChannel.HttpChannelAsyncRequest.CompleteGetResponse(IAsyncResult result)"},"StackTrace":"   at System.ServiceModel.AsyncResult.End[TAsyncResult](IAsyncResult result)\r\n   at System.ServiceModel.Channels.ServiceChannel.EndCall(String action, Object[] outs, IAsyncResult result)\r\n   at System.ServiceModel.ClientBase`1.ChannelBase`1.EndInvoke(String methodName, Object[] args, IAsyncResult result)\r\n   at CCWFM.CRUDManagerService.CRUD_ManagerServiceClient.CRUD_ManagerServiceClientChannel.EndFamilyCategory_GetTblFabricDesign(Int32& fullCount, IAsyncResult result)\r\n   at CCWFM.CRUDManagerService.CRUD_ManagerServiceClient.CCWFM.CRUDManagerService.CRUD_ManagerService.EndFamilyCategory_GetTblFabricDesign(Int32& fullCount, IAsyncResult result)\r\n   at CCWFM.CRUDManagerService.CRUD_ManagerServiceClient.OnEndFamilyCategory_GetTblFabricDesign(IAsyncResult result)\r\n   at System.ServiceModel.ClientBase`1.OnAsyncCallCompleted(IAsyncResult result)"},"StackTrace":"   at System.ComponentModel.AsyncCompletedEventArgs.RaiseExceptionIfNecessary()\r\n   at CCWFM.CRUDManagerService.FamilyCategory_GetTblFabricDesignCompletedEventArgs.get_Result()\r\n   at CCWFM.ViewModel.OGViewModels.StyleHeaderViewModel.<GetDesign>b__83_0(Object s, FamilyCategory_GetTblFabricDesignCompletedEventArgs sv)\r\n   at CCWFM.CRUDManagerService.CRUD_ManagerServiceClient.OnFamilyCategory_GetTblFabricDesignCompleted(Object state)"}
2021-09-19 14:25:39.5639|INFO|Info|User Iserial:1144
2021-09-19 14:25:39.5639|ERROR|Info|{"Message":"An exception occurred during the operation, making the result invalid.  Check InnerException for exception details.","Data":{},"InnerException":{"Message":"The HTTP request to 'http://localhost:18559/Service/CRUD_ManagerService.svc' has exceeded the allotted timeout. The time allotted to this operation may have been a portion of a longer timeout.","Data":{},"InnerException":{"Status":6,"Response":null,"Message":"","Data":{},"InnerException":{"Status":6,"Response":null,"Message":"","Data":{},"InnerException":null,"StackTrace":"   at System.Net.Browser.BrowserHttpWebRequest.InternalEndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Browser.BrowserHttpWebRequest.<>c__DisplayClassa.<EndGetResponse>b__9(Object sendState)\r\n   at System.Net.Browser.AsyncHelper.<>c__DisplayClass4.<BeginOnUI>b__0(Object sendState)"},"StackTrace":"   at System.Net.Browser.AsyncHelper.BeginOnUI(SendOrPostCallback beginMethod, Object state)\r\n   at System.Net.Browser.BrowserHttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.ServiceModel.Channels.HttpChannelFactory.HttpRequestChannel.HttpChannelAsyncRequest.CompleteGetResponse(IAsyncResult result)"},"StackTrace":"   at System.ServiceModel.AsyncResult.End[TAsyncResult](IAsyncResult result)\r\n   at System.ServiceModel.Channels.ServiceChannel.EndCall(String action, Object[] outs, IAsyncResult result)\r\n   at System.ServiceModel.ClientBase`1.ChannelBase`1.EndInvoke(String methodName, Object[] args, IAsyncResult result)\r\n   at CCWFM.CRUDManagerService.CRUD_ManagerServiceClient.CRUD_ManagerServiceClientChannel.EndGetGeneric(IAsyncResult result)\r\n   at CCWFM.CRUDManagerService.CRUD_ManagerServiceClient.CCWFM.CRUDManagerService.CRUD_ManagerService.EndGetGeneric(IAsyncResult result)\r\n   at CCWFM.CRUDManagerService.CRUD_ManagerServiceClient.OnEndGetGeneric(IAsyncResult result)\r\n   at System.ServiceModel.ClientBase`1.OnAsyncCallCompleted(IAsyncResult result)"},"StackTrace":"   at System.ComponentModel.AsyncCompletedEventArgs.RaiseExceptionIfNecessary()\r\n   at CCWFM.CRUDManagerService.GetGenericCompletedEventArgs.get_Result()\r\n   at CCWFM.ViewModel.OGViewModels.StyleHeaderViewModel.<.ctor>b__32_4(Object s, GetGenericCompletedEventArgs sv)\r\n   at CCWFM.CRUDManagerService.CRUD_ManagerServiceClient.OnGetGenericCompleted(Object state)"}
2021-09-19 14:25:39.6129|INFO|Info|User Iserial:1144
2021-09-19 14:25:39.6129|ERROR|Info|{"Message":"An exception occurred during the operation, making the result invalid.  Check InnerException for exception details.","Data":{},"InnerException":{"Message":"The HTTP request to 'http://localhost:18559/Service/CRUD_ManagerService.svc' has exceeded the allotted timeout. The time allotted to this operation may have been a portion of a longer timeout.","Data":{},"InnerException":{"Status":6,"Response":null,"Message":"","Data":{},"InnerException":{"Status":6,"Response":null,"Message":"","Data":{},"InnerException":null,"StackTrace":"   at System.Net.Browser.BrowserHttpWebRequest.InternalEndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Browser.BrowserHttpWebRequest.<>c__DisplayClassa.<EndGetResponse>b__9(Object sendState)\r\n   at System.Net.Browser.AsyncHelper.<>c__DisplayClass4.<BeginOnUI>b__0(Object sendState)"},"StackTrace":"   at System.Net.Browser.AsyncHelper.BeginOnUI(SendOrPostCallback beginMethod, Object state)\r\n   at System.Net.Browser.BrowserHttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.ServiceModel.Channels.HttpChannelFactory.HttpRequestChannel.HttpChannelAsyncRequest.CompleteGetResponse(IAsyncResult result)"},"StackTrace":"   at System.ServiceModel.AsyncResult.End[TAsyncResult](IAsyncResult result)\r\n   at System.ServiceModel.Channels.ServiceChannel.EndCall(String action, Object[] outs, IAsyncResult result)\r\n   at System.ServiceModel.ClientBase`1.ChannelBase`1.EndInvoke(String methodName, Object[] args, IAsyncResult result)\r\n   at CCWFM.CRUDManagerService.CRUD_ManagerServiceClient.CRUD_ManagerServiceClientChannel.EndGetRetailTblChainSetting(IAsyncResult result)\r\n   at CCWFM.CRUDManagerService.CRUD_ManagerServiceClient.CCWFM.CRUDManagerService.CRUD_ManagerService.EndGetRetailTblChainSetting(IAsyncResult result)\r\n   at CCWFM.CRUDManagerService.CRUD_ManagerServiceClient.OnEndGetRetailTblChainSetting(IAsyncResult result)\r\n   at System.ServiceModel.ClientBase`1.OnAsyncCallCompleted(IAsyncResult result)"},"StackTrace":"   at System.ComponentModel.AsyncCompletedEventArgs.RaiseExceptionIfNecessary()\r\n   at CCWFM.CRUDManagerService.GetRetailTblChainSettingCompletedEventArgs.get_Result()\r\n   at CCWFM.ViewModel.OGViewModels.StyleHeaderViewModel.<.ctor>b__32_132(Object s, GetRetailTblChainSettingCompletedEventArgs sv)\r\n   at CCWFM.CRUDManagerService.CRUD_ManagerServiceClient.OnGetRetailTblChainSettingCompleted(Object state)"}
2021-09-19 14:25:40.0646|INFO|Info|User Iserial:1144
2021-09-19 14:25:40.0646|ERROR|Info|{"Message":"An exception occurred during the operation, making the result invalid.  Check InnerException for exception details.","Data":{},"InnerException":{"Message":"The HTTP request to 'http://localhost:18559/Service/CRUD_ManagerService.svc' has exceeded the allotted timeout. The time allotted to this operation may have been a portion of a longer timeout.","Data":{},"InnerException":{"Status":6,"Response":null,"Message":"","Data":{},"InnerException":{"Status":6,"Response":null,"Message":"","Data":{},"InnerException":null,"StackTrace":"   at System.Net.Browser.BrowserHttpWebRequest.InternalEndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Browser.BrowserHttpWebRequest.<>c__DisplayClassa.<EndGetResponse>b__9(Object sendState)\r\n   at System.Net.Browser.AsyncHelper.<>c__DisplayClass4.<BeginOnUI>b__0(Object sendState)"},"StackTrace":"   at System.Net.Browser.AsyncHelper.BeginOnUI(SendOrPostCallback beginMethod, Object state)\r\n   at System.Net.Browser.BrowserHttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.ServiceModel.Channels.HttpChannelFactory.HttpRequestChannel.HttpChannelAsyncRequest.CompleteGetResponse(IAsyncResult result)"},"StackTrace":"   at System.ServiceModel.AsyncResult.End[TAsyncResult](IAsyncResult result)\r\n   at System.ServiceModel.Channels.ServiceChannel.EndCall(String action, Object[] outs, IAsyncResult result)\r\n   at System.ServiceModel.ClientBase`1.ChannelBase`1.EndInvoke(String methodName, Object[] args, IAsyncResult result)\r\n   at CCWFM.CRUDManagerService.CRUD_ManagerServiceClient.CRUD_ManagerServiceClientChannel.EndGetAllSeasons(IAsyncResult result)\r\n   at CCWFM.CRUDManagerService.CRUD_ManagerServiceClient.CCWFM.CRUDManagerService.CRUD_ManagerService.EndGetAllSeasons(IAsyncResult result)\r\n   at CCWFM.CRUDManagerService.CRUD_ManagerServiceClient.OnEndGetAllSeasons(IAsyncResult result)\r\n   at System.ServiceModel.ClientBase`1.OnAsyncCallCompleted(IAsyncResult result)"},"StackTrace":"   at System.ComponentModel.AsyncCompletedEventArgs.RaiseExceptionIfNecessary()\r\n   at CCWFM.CRUDManagerService.GetAllSeasonsCompletedEventArgs.get_Result()\r\n   at CCWFM.ViewModel.OGViewModels.StyleHeaderViewModel.<GetSeason>b__82_0(Object s, GetAllSeasonsCompletedEventArgs sv)\r\n   at CCWFM.CRUDManagerService.CRUD_ManagerServiceClient.OnGetAllSeasonsCompleted(Object state)"}
2021-09-19 14:25:41.6055|INFO|Info|User Iserial:1144
2021-09-19 14:25:41.6055|ERROR|Info|{"Message":"An exception occurred during the operation, making the result invalid.  Check InnerException for exception details.","Data":{},"InnerException":{"Message":"The HTTP request to 'http://localhost:18559/Service/CRUD_ManagerService.svc' has exceeded the allotted timeout. The time allotted to this operation may have been a portion of a longer timeout.","Data":{},"InnerException":{"Status":6,"Response":null,"Message":"","Data":{},"InnerException":{"Status":6,"Response":null,"Message":"","Data":{},"InnerException":null,"StackTrace":"   at System.Net.Browser.BrowserHttpWebRequest.InternalEndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Browser.BrowserHttpWebRequest.<>c__DisplayClassa.<EndGetResponse>b__9(Object sendState)\r\n   at System.Net.Browser.AsyncHelper.<>c__DisplayClass4.<BeginOnUI>b__0(Object sendState)"},"StackTrace":"   at System.Net.Browser.AsyncHelper.BeginOnUI(SendOrPostCallback beginMethod, Object state)\r\n   at System.Net.Browser.BrowserHttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.ServiceModel.Channels.HttpChannelFactory.HttpRequestChannel.HttpChannelAsyncRequest.CompleteGetResponse(IAsyncResult result)"},"StackTrace":"   at System.ServiceModel.AsyncResult.End[TAsyncResult](IAsyncResult result)\r\n   at System.ServiceModel.Channels.ServiceChannel.EndCall(String action, Object[] outs, IAsyncResult result)\r\n   at System.ServiceModel.ClientBase`1.ChannelBase`1.EndInvoke(String methodName, Object[] args, IAsyncResult result)\r\n   at CCWFM.CRUDManagerService.CRUD_ManagerServiceClient.CRUD_ManagerServiceClientChannel.EndGetGeneric(IAsyncResult result)\r\n   at CCWFM.CRUDManagerService.CRUD_ManagerServiceClient.CCWFM.CRUDManagerService.CRUD_ManagerService.EndGetGeneric(IAsyncResult result)\r\n   at CCWFM.CRUDManagerService.CRUD_ManagerServiceClient.OnEndGetGeneric(IAsyncResult result)\r\n   at System.ServiceModel.ClientBase`1.OnAsyncCallCompleted(IAsyncResult result)"},"StackTrace":"   at System.ComponentModel.AsyncCompletedEventArgs.RaiseExceptionIfNecessary()\r\n   at CCWFM.CRUDManagerService.GetGenericCompletedEventArgs.get_Result()\r\n   at CCWFM.ViewModel.OGViewModels.StyleHeaderViewModel.<.ctor>b__32_5(Object s, GetGenericCompletedEventArgs sv)\r\n   at CCWFM.CRUDManagerService.CRUD_ManagerServiceClient.OnGetGenericCompleted(Object state)"}
2021-09-19 14:31:44.1638|INFO|Info|User Iserial:1144
2021-09-19 14:31:44.1638|ERROR|Info|{"Message":"An exception occurred during the operation, making the result invalid.  Check InnerException for exception details.","Data":{},"InnerException":{"Message":"The HTTP request to 'http://localhost:18559/Service/CRUD_ManagerService.svc' has exceeded the allotted timeout. The time allotted to this operation may have been a portion of a longer timeout.","Data":{},"InnerException":{"Status":6,"Response":null,"Message":"","Data":{},"InnerException":{"Status":6,"Response":null,"Message":"","Data":{},"InnerException":null,"StackTrace":"   at System.Net.Browser.BrowserHttpWebRequest.InternalEndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Browser.BrowserHttpWebRequest.<>c__DisplayClassa.<EndGetResponse>b__9(Object sendState)\r\n   at System.Net.Browser.AsyncHelper.<>c__DisplayClass4.<BeginOnUI>b__0(Object sendState)"},"StackTrace":"   at System.Net.Browser.AsyncHelper.BeginOnUI(SendOrPostCallback beginMethod, Object state)\r\n   at System.Net.Browser.BrowserHttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.ServiceModel.Channels.HttpChannelFactory.HttpRequestChannel.HttpChannelAsyncRequest.CompleteGetResponse(IAsyncResult result)"},"StackTrace":"   at System.ServiceModel.AsyncResult.End[TAsyncResult](IAsyncResult result)\r\n   at System.ServiceModel.Channels.ServiceChannel.EndCall(String action, Object[] outs, IAsyncResult result)\r\n   at System.ServiceModel.ClientBase`1.ChannelBase`1.EndInvoke(String methodName, Object[] args, IAsyncResult result)\r\n   at CCWFM.CRUDManagerService.CRUD_ManagerServiceClient.CRUD_ManagerServiceClientChannel.EndGetTblSizeGroup(Int32& fullCount, IAsyncResult result)\r\n   at CCWFM.CRUDManagerService.CRUD_ManagerServiceClient.CCWFM.CRUDManagerService.CRUD_ManagerService.EndGetTblSizeGroup(Int32& fullCount, IAsyncResult result)\r\n   at CCWFM.CRUDManagerService.CRUD_ManagerServiceClient.OnEndGetTblSizeGroup(IAsyncResult result)\r\n   at System.ServiceModel.ClientBase`1.OnAsyncCallCompleted(IAsyncResult result)"},"StackTrace":"   at System.ComponentModel.AsyncCompletedEventArgs.RaiseExceptionIfNecessary()\r\n   at CCWFM.CRUDManagerService.GetTblSizeGroupCompletedEventArgs.get_Result()\r\n   at CCWFM.ViewModel.OGViewModels.StyleHeaderViewModel.<.ctor>b__32_43(Object s, GetTblSizeGroupCompletedEventArgs sv)\r\n   at CCWFM.CRUDManagerService.CRUD_ManagerServiceClient.OnGetTblSizeGroupCompleted(Object state)"}
2021-09-19 14:31:44.6777|INFO|Info|User Iserial:1144
2021-09-19 14:31:44.6777|ERROR|Info|{"Message":"An exception occurred during the operation, making the result invalid.  Check InnerException for exception details.","Data":{},"InnerException":{"Message":"The HTTP request to 'http://localhost:18559/Service/CRUD_ManagerService.svc' has exceeded the allotted timeout. The time allotted to this operation may have been a portion of a longer timeout.","Data":{},"InnerException":{"Status":6,"Response":null,"Message":"","Data":{},"InnerException":{"Status":6,"Response":null,"Message":"","Data":{},"InnerException":null,"StackTrace":"   at System.Net.Browser.BrowserHttpWebRequest.InternalEndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Browser.BrowserHttpWebRequest.<>c__DisplayClassa.<EndGetResponse>b__9(Object sendState)\r\n   at System.Net.Browser.AsyncHelper.<>c__DisplayClass4.<BeginOnUI>b__0(Object sendState)"},"StackTrace":"   at System.Net.Browser.AsyncHelper.BeginOnUI(SendOrPostCallback beginMethod, Object state)\r\n   at System.Net.Browser.BrowserHttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.ServiceModel.Channels.HttpChannelFactory.HttpRequestChannel.HttpChannelAsyncRequest.CompleteGetResponse(IAsyncResult result)"},"StackTrace":"   at System.ServiceModel.AsyncResult.End[TAsyncResult](IAsyncResult result)\r\n   at System.ServiceModel.Channels.ServiceChannel.EndCall(String action, Object[] outs, IAsyncResult result)\r\n   at System.ServiceModel.ClientBase`1.ChannelBase`1.EndInvoke(String methodName, Object[] args, IAsyncResult result)\r\n   at CCWFM.CRUDManagerService.CRUD_ManagerServiceClient.CRUD_ManagerServiceClientChannel.EndGetItemsPermissionsByParent(IAsyncResult result)\r\n   at CCWFM.CRUDManagerService.CRUD_ManagerServiceClient.CCWFM.CRUDManagerService.CRUD_ManagerService.EndGetItemsPermissionsByParent(IAsyncResult result)\r\n   at CCWFM.CRUDManagerService.CRUD_ManagerServiceClient.OnEndGetItemsPermissionsByParent(IAsyncResult result)\r\n   at System.ServiceModel.ClientBase`1.OnAsyncCallCompleted(IAsyncResult result)"},"StackTrace":"   at System.ComponentModel.AsyncCompletedEventArgs.RaiseExceptionIfNecessary()\r\n   at CCWFM.CRUDManagerService.GetItemsPermissionsByParentCompletedEventArgs.get_Result()\r\n   at CCWFM.ViewModel.ViewModelBase.<GetCustomePermissions>b__127_0(Object s, GetItemsPermissionsByParentCompletedEventArgs e)\r\n   at CCWFM.CRUDManagerService.CRUD_ManagerServiceClient.OnGetItemsPermissionsByParentCompleted(Object state)"}
2021-09-19 14:31:45.1787|INFO|Info|User Iserial:1144
2021-09-19 14:31:45.1787|ERROR|Info|{"Message":"An exception occurred during the operation, making the result invalid.  Check InnerException for exception details.","Data":{},"InnerException":{"Message":"The HTTP request to 'http://localhost:18559/Service/CRUD_ManagerService.svc' has exceeded the allotted timeout. The time allotted to this operation may have been a portion of a longer timeout.","Data":{},"InnerException":{"Status":6,"Response":null,"Message":"","Data":{},"InnerException":{"Status":6,"Response":null,"Message":"","Data":{},"InnerException":null,"StackTrace":"   at System.Net.Browser.BrowserHttpWebRequest.InternalEndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Browser.BrowserHttpWebRequest.<>c__DisplayClassa.<EndGetResponse>b__9(Object sendState)\r\n   at System.Net.Browser.AsyncHelper.<>c__DisplayClass4.<BeginOnUI>b__0(Object sendState)"},"StackTrace":"   at System.Net.Browser.AsyncHelper.BeginOnUI(SendOrPostCallback beginMethod, Object state)\r\n   at System.Net.Browser.BrowserHttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.ServiceModel.Channels.HttpChannelFactory.HttpRequestChannel.HttpChannelAsyncRequest.CompleteGetResponse(IAsyncResult result)"},"StackTrace":"   at System.ServiceModel.AsyncResult.End[TAsyncResult](IAsyncResult result)\r\n   at System.ServiceModel.Channels.ServiceChannel.EndCall(String action, Object[] outs, IAsyncResult result)\r\n   at System.ServiceModel.ClientBase`1.ChannelBase`1.EndInvoke(String methodName, Object[] args, IAsyncResult result)\r\n   at CCWFM.CRUDManagerService.CRUD_ManagerServiceClient.CRUD_ManagerServiceClientChannel.EndGetGeneric(IAsyncResult result)\r\n   at CCWFM.CRUDManagerService.CRUD_ManagerServiceClient.CCWFM.CRUDManagerService.CRUD_ManagerService.EndGetGeneric(IAsyncResult result)\r\n   at CCWFM.CRUDManagerService.CRUD_ManagerServiceClient.OnEndGetGeneric(IAsyncResult result)\r\n   at System.ServiceModel.ClientBase`1.OnAsyncCallCompleted(IAsyncResult result)"},"StackTrace":"   at System.ComponentModel.AsyncCompletedEventArgs.RaiseExceptionIfNecessary()\r\n   at CCWFM.CRUDManagerService.GetGenericCompletedEventArgs.get_Result()\r\n   at CCWFM.ViewModel.OGViewModels.StyleHeaderViewModel.<.ctor>b__32_42(Object s, GetGenericCompletedEventArgs sv)\r\n   at CCWFM.CRUDManagerService.CRUD_ManagerServiceClient.OnGetGenericCompleted(Object state)"}
2021-09-19 14:31:45.6952|INFO|Info|User Iserial:1144
2021-09-19 14:31:45.6952|ERROR|Info|{"Message":"An exception occurred during the operation, making the result invalid.  Check InnerException for exception details.","Data":{},"InnerException":{"Message":"The HTTP request to 'http://localhost:18559/Service/CRUD_ManagerService.svc' has exceeded the allotted timeout. The time allotted to this operation may have been a portion of a longer timeout.","Data":{},"InnerException":{"Status":6,"Response":null,"Message":"","Data":{},"InnerException":{"Status":6,"Response":null,"Message":"","Data":{},"InnerException":null,"StackTrace":"   at System.Net.Browser.BrowserHttpWebRequest.InternalEndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Browser.BrowserHttpWebRequest.<>c__DisplayClassa.<EndGetResponse>b__9(Object sendState)\r\n   at System.Net.Browser.AsyncHelper.<>c__DisplayClass4.<BeginOnUI>b__0(Object sendState)"},"StackTrace":"   at System.Net.Browser.AsyncHelper.BeginOnUI(SendOrPostCallback beginMethod, Object state)\r\n   at System.Net.Browser.BrowserHttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.ServiceModel.Channels.HttpChannelFactory.HttpRequestChannel.HttpChannelAsyncRequest.CompleteGetResponse(IAsyncResult result)"},"StackTrace":"   at System.ServiceModel.AsyncResult.End[TAsyncResult](IAsyncResult result)\r\n   at System.ServiceModel.Channels.ServiceChannel.EndCall(String action, Object[] outs, IAsyncResult result)\r\n   at System.ServiceModel.ClientBase`1.ChannelBase`1.EndInvoke(String methodName, Object[] args, IAsyncResult result)\r\n   at CCWFM.CRUDManagerService.CRUD_ManagerServiceClient.CRUD_ManagerServiceClientChannel.EndGetAllSeasons(IAsyncResult result)\r\n   at CCWFM.CRUDManagerService.CRUD_ManagerServiceClient.CCWFM.CRUDManagerService.CRUD_ManagerService.EndGetAllSeasons(IAsyncResult result)\r\n   at CCWFM.CRUDManagerService.CRUD_ManagerServiceClient.OnEndGetAllSeasons(IAsyncResult result)\r\n   at System.ServiceModel.ClientBase`1.OnAsyncCallCompleted(IAsyncResult result)"},"StackTrace":"   at System.ComponentModel.AsyncCompletedEventArgs.RaiseExceptionIfNecessary()\r\n   at CCWFM.CRUDManagerService.GetAllSeasonsCompletedEventArgs.get_Result()\r\n   at CCWFM.ViewModel.OGViewModels.StyleHeaderViewModel.<GetSeason>b__82_0(Object s, GetAllSeasonsCompletedEventArgs sv)\r\n   at CCWFM.CRUDManagerService.CRUD_ManagerServiceClient.OnGetAllSeasonsCompleted(Object state)"}
